__tests__/integration.test.js

```diff
@@ -35,14 +33,60 @@ describe("GET: /api/invalid_endpoint", () => {
   });
 });
 describe("GET: /api", () => {
-  test("Should return a JSON object for all articles", () => {
+  test("Should return a JSON object for all endpoints", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoint).toEqual(expectedContents);
+      });
+  });
+});
+describe("GET: /api/articles/:article_id", () => {
+  test("GET: responds with a 200 status code and a nested object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toBeInstanceOf(Object)
+        expect(body.articles).toBeInstanceOf(Object)
+      })
+  });
+  test("GET: responds with an object with the expected properties", () => {
     return request(app)
-    .get("/api/")
+    .get("/api/articles/1")
     .expect(200)
     .then(({body}) => {
-        expect(body.endpoint).toEqual(expectedContents)
-    })
+      const { articles } = body
+      articles.forEach((article) => {
```

Comment:
We shouldn't need to loop through the response, it should just be a single article object